File: src/BreakfastProvider.Api/Models/Responses/InventoryItemResponse.cs
namespace BreakfastProvider.Api.Models.Responses;

public record InventoryItemResponse
{
    public int Id { get; init; }
    public string Name { get; init; } = string.Empty;
    public string Category { get; init; } = string.Empty;
    public decimal Quantity { get; init; }
    public string Unit { get; init; } = string.Empty;
    public decimal ReorderLevel { get; init; }
    public DateTime LastRestockedAt { get; init; }
    public DateTime CreatedAt { get; init; }
}

File: java/breakfast-provider-app/src/main/java/io/lemonlion/breakfast/model/response/InventoryItemResponse.java
package io.lemonlion.breakfast.model.response;

import java.math.BigDecimal;
import java.time.Instant;

/** Twin of C# {@code InventoryItemResponse}. */
public record InventoryItemResponse(
        int id,
        String name,
        String category,
        BigDecimal quantity,
        String unit,
        BigDecimal reorderLevel,
        Instant lastRestockedAt,
        Instant createdAt) {
}
